scripts/generate-docs/OptionsDoc.ts

```diff
@@ -195,49 +195,15 @@ export default class OptionsDoc extends Doc {
           ),
         ).then(beautified => {
           builder.header("Examples", 2);
-          builder.append(
-            '<div class="input-group languages-select"><div class="input-group-prepend">',
-          );
-          builder.append(
-            '<label class="input-group-text" for="languages-select">Select Language:</label>',
-          );
-          builder.append(
-            '</div><select class="custom-select" id="languages-select">',
-          );
-          let defaultDisplay: string;
-          let isDefault: boolean = true;
-          this.languages.forEach(language => {
-            const example = examplesForLanguages[language.name];
-            builder.append(
-              `<option ${
-                example && isDefault ? 'selected="selected"' : ""
-              } data-text="${
-                language.name
-              }" value="${language.name
```

Comment:
@stevenzeck Before we had the direct links to select the appropriate tab: https://unibeautify.com/docs/option-end-with-newline.html?language=html

Now the similar link will no longer select the code tab: https://deploy-preview-211--unibeautify.netlify.com/docs/option-end-with-newline.html?language=html

I do want to continue to support this feature.